lib/tasks/onetime.rake

```diff
@@ -0,0 +1,37 @@
+namespace :onetime do
+  def table_exists?
+    ActiveRecord::Base.connection.table_exists? 'onetime_scripts'
+  end
+
+  desc 'Checks if the required table exists'
+  task check_table_exists: :environment  do
+    if table_exists?
+      puts "The table onetime_scripts exists"
+    else
+      raise "The table onetime_scripts doesn't exist\nTo create the table run 'rake onetime:install'\n\n"
```

Comment:
I was hoping to do this automatically. You never have to think about creating the schema_migrations table when running rake db:migrate, it just checks to see if it exists and creates it if need be